proto/eth/v1/custom.go

```diff
@@ -0,0 +1,38 @@
+package v1
+
+import (
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	ethrpc "github.com/prysmaticlabs/prysm/v3/beacon-chain/rpc/apimiddleware"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"strconv"
+	"time"
+)
+
+func NewSyncAggregateFromJSON(syncAggregate *ethrpc.SyncAggregateJson) *SyncAggregate {
+	return &SyncAggregate{
+		SyncCommitteeBits:      hexutil.MustDecode(syncAggregate.SyncCommitteeBits),
```

Comment:
I think this is a uint512 (as sync committee size is 512).   
So same question as above.

Reply:
ditto